### PR TITLE
 second commit

### PR DESCRIPTION
**Pull request by AlbertAndrei**
dfghjkl

### CODE REVIEW

File: src/main/java/Darts.java [hunk 1]
```diff
@@ -8,4 +8,51 @@
     private boolean isFinished = false;
     private int turn = 1;
     private int lastTurnScore = score;
+
+    public void darts(int scoreTemp) {
```
Comment (octavianCiobanuca) [inline src/main/java/Darts.java:14]: asddsasd

Comment (AlbertAndrei) [reply src/main/java/Darts.java:14]: taci ca e bun asa

Comment (octavianCiobanuca) [reply src/main/java/Darts.java:14]: asd

Comment (octavianCiobanuca) [reply src/main/java/Darts.java:14]: bhjv

**Review by octavianCiobanuca — DISMISSED** [review]

**Pull request merged**